pycm/pycm_param.py

```diff
@@ -46,87 +46,76 @@
-
-AVERAGE_WEIGHT_ERROR = "The weight type must be dictionary and also must be specified for all of the classes."
-
+PROBABILITY_SIZE_ERROR = "All elements of the probability vector must have the same length and match the number of classes."
+PROBABILITY_TYPE_ERROR = "Probability vector elements must be numeric."
+PROBABILITY_SUM_ERROR = "The sum of the probability values must equal 1."
+THRESHOLDS_NUMBER_ERROR = "The number of thresholds must be at least 2."
+THRESHOLDS_TYPE_ERROR = "Thresholds must be provided as a list or a NumPy array."
+THRESHOLDS_NUMERIC_ERROR = "Thresholds must contain only numeric values."
+CLASSES_TYPE_ERROR = "Classes must be provided as a list."
+CLASSES_MATCH_ERROR = "The provided classes do not match the actual vector."
+MATRIX_CLASS_TYPE_ERROR = "All input matrix classes must be of the same type."
+MATRIX_FORMAT_ERROR = "Invalid input confusion matrix format."
+MAPPING_FORMAT_ERROR = "Invalid mapping format."
+MAPPING_CLASS_NAME_ERROR = "Invalid mapping class names."
+SEABORN_PLOT_LIBRARY_ERROR = "Failed to import seaborn module. Please install it using: `pip install seaborn`."
+MATPLOTLIB_PLOT_LIBRARY_ERROR = "Failed to import matplotlib module. Please install it using: `pip install matplotlib`."
+PLOT_COLORS_CLASS_MISMATCH_ERROR = "The number of colors does not match the number of classes."
+PLOT_MARKERS_CLASS_MISMATCH_ERROR = "The number of markers does not match the number of classes."
+VECTOR_TYPE_ERROR = "Input vectors must be provided as a list or a NumPy array."
+VECTOR_SIZE_ERROR = "Input vectors must have the same length."
+VECTOR_EMPTY_ERROR = "Input vectors must not be empty."
+VECTOR_ONLY_ERROR = "This option is only available in vector mode."
+VECTOR_UNIQUE_CLASS_ERROR = "The classes list must contain unique labels with no duplicates."
+NOT_ALL_SET_VECTOR_ERROR = "Failed to extract classes from input. Input vectors should be a list of sets with unified types."
+CLASS_NUMBER_ERROR = "The number of classes must be at least 2."
+METRICS_OFF_ERROR = "This method cannot be executed when `metrics_off=True`."
+CLASSES_ERROR = "The specified classes are not a subset of the matrix's classes."
+COMPARE_FORMAT_ERROR = "Input must be provided as a dictionary."
+CLASSES_LENGTH_ERROR = "The length of the classes does not match the length of the array."
+AREA_METHOD_ERROR = "The integral method must be either 'trapezoidal' or 'midpoint'."
+
+VECTOR_INDEX_ERROR = "Index is out of range for the given vector."
+INVALID_CLASS_NAME_ERROR = "The specified class name is not among the problem's classes."
+
+COMPARE_TYPE_ERROR = "Input must be an instance of pycm.ConfusionMatrix."
+COMPARE_DOMAIN_ERROR = "All ConfusionMatrix objects must have the same domain (same sample size and number of classes)."
+COMPARE_NUMBER_ERROR = "At least 2 confusion matrices are required for comparison."
+
+COMPARE_CLASS_WEIGHT_ERROR = "class_weight must be a dictionary and specified for all classes."
```

Comment:
I suggest using backticks (`` ` ``) around all parameter names. For example:

```
`class_weight`
```
or 
```
`class_benchmark_weight`
```